src/main/java/io/appium/java_client/pagefactory/bys/builder/AppiumByBuilder.java

```diff
@@ -96,7 +96,7 @@ private static String getFilledValue(Annotation mobileBy) {
             }
 
             try {
-                String strategyParameter = value.invoke(mobileBy, new Object[] {}).toString();
+                String strategyParameter = value.invoke(mobileBy).toString();
```

Comment:
does this call return the same result as the previous one?

Reply:
yes, the method signature is:
```java
java.lang.reflect.Method.invoke(Object, Object...)
```
so it's not required to pass an empty array as varargs